src/main/java/batman/tasks/TaskList.java

```diff
@@ -0,0 +1,186 @@
+package batman.tasks;
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+import batman.exception.DukeException;
+import batman.exception.Error;
+
+public class TaskList {
+
+    private static ArrayList<Task> tasks;
+
+    public TaskList() {
+        tasks = new ArrayList<>();
+    }
+
+    public Task get(int index) {
+        return tasks.get(index);
+    }
+
+    public int getSize() {
+        return tasks.size();
+    }
+
+    public ArrayList<Task> getTaskList() {
+        return tasks;
+    }
+
+    /**
+     * Adds tasks depending on what type of task it is.
+     *
+     * @param command An array of strings containing the
+     *                details of a task.
+     */
```

Comment:
Shouldn't there be a @return parameter in the JavaDoc comments? 
```suggestion
     @return the response message.
     */
```